app/Http/Controllers/Controller.php

```
@@ -98,58 +98,81 @@ public function forgot_password(Request $request)
+            $user->teams()->attach($team->id, ['role' => $invitation->role]);
+        }
+        $invitation->delete();
 
-            if (Hash::check($password, $user->password)) {
-                $team = $invitation->team;
-                if (! $user->teams()->where('team_id', $team->id)->exists()) {
-                    $user->teams()->attach($team->id, ['role' => $invitation->role]);
-                }
-                $invitation->delete();
+        $user->forceFill([
+            'password' => Hash::make(Str::random(64)),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Hash' in method 'acceptLink'. (undefined)

(StaticAccess)